metabrainz/templates/index/about.html

```diff
@@ -1,66 +1,67 @@
 {% extends 'base.html' %}
 
-{% block title %}About - MetaBrainz Foundation{% endblock %}
+{% block title %}{{ _('About') }} - MetaBrainz Foundation{% endblock %}
 
 {% block content %}
-  <h1 class="page-title">About the MetaBrainz Foundation</h1>
+  <h1 class="page-title">{{ _('About the MetaBrainz Foundation') }}</h1>
 
   <div class="row">
     <div class="col-sm-9">
       <p>
-        The MetaBrainz Foundation is a 501(c)(3) tax-exempt non-profit founded in
+        {{ _('The MetaBrainz Foundation is a 501(c)(3) tax-exempt non-profit founded in
         December 2004 and based in San Luis Obispo, California, that believes in
-        free, open access to data.
+        free, open access to data.') }}
       </p>
       <p>
-        The MetaBrainz Foundation has been set up to build community maintained
+        {{ _('The MetaBrainz Foundation has been set up to build community maintained
         databases and make them available in the public domain or under Creative
         Commons licenses. Our data is mostly gathered by volunteers, and verified
         by a voting system to make sure it is consistent and correct. All
         non-commercial use is free, but commercial users are asked to pay for
         regular updates of the data in order to help fund the project. We
         encourage all data users to contribute to the data gathering process so
-        that our data can be as comprehensive as possible.
+        that our data can be as comprehensive as possible.') }}
       </p>
       <p>
-        The MetaBrainz Foundation believes in <a href="{{ url_for('financial_reports.index') }}">
-        transparent finances</a> and has several <a href="{{ url_for('users.supporters_list') }}">
+        {{ _('The MetaBrainz Foundation believes in <a href="%(url_1)s">
+        transparent finances</a> and has several <a href="%(url_2)s">
         commercial supporters</a> who pay for timely and convenient access to
         data. The MetaBrainz Foundation is also supported by a number of
-        <a href="{{ url_for('index.sponsors') }}">sponsors</a> who provide
-        funds in order for the foundation to accomplish its goals.
+        <a href="%(url_3)s">sponsors</a> who provide
+        funds in order for the foundation to accomplish its goals.',
+        url_1=url_for('financial_reports.index'), url_2=url_for('users.supporters_list'), url_3=url_for('index.sponsors')) }}
```

Comment:
The names of the variables for the URLs can be more descriptive. Like `supporters_url` instead of `url_1`.